release-notes.md

```diff
@@ -160,6 +160,12 @@ This document contains release notes per zib, indicating differences with their
 * New concept Indication mapped on the extension `NutritionOrder.extension:indication`.
 * The comment extension has been replaced by a mapping to `NutritionOrder.note.text`.
 
+## zib-OutcomeOfCare
+* The code on `DiagnosticReport.code` has been changed to 423100009 and the system value to http://snomed.info/sct.
+* Removed MeasurementValue mapping because this element does not exist anymore.
+* Element HealthCondition::FunctionalOrMentalStatus with ID NL-CM:13.4.3 is mapped on `DiagnosticReport.result` instead of on its own slice.
```

Comment:
Shouldn't we keep this element on a separate slice so that it's possible to add other references next to the one defined in the zib?